sql/parser/type_check_test.go

```diff
@@ -26,6 +26,12 @@ import (
 
 func TestTypeCheckExpr(t *testing.T) {
 	testData := []string{
+		`NULL + 1`,
+		`NULL + 1.1`,
+		`NULL + '2006-09-23'::date`,
+		`NULL + '1h'::interval`,
+		`NULL + 'hello'`,
+		`NULL + 'hello'::bytes`,
```

Comment:
I wrote the eval test that way.

On Thu, Oct 15, 2015 at 2:55 PM, Peter Mattis <notifications@github.com>
wrote:

> In sql/parser/type_check_test.go
> <https://github.com/cockroachdb/cockroach/pull/2848#discussion_r42165171>:
>
> > @@ -26,6 +26,12 @@ import (
> >
> >  func TestTypeCheckExpr(t *testing.T) {
> >  	testData := []string{
> > +		`NULL + 1`,
> > +		`NULL + 1.1`,
> > +		`NULL + '2006-09-23'::date`,
> > +		`NULL + '1h'::interval`,
> > +		`NULL + 'hello'`,
> > +		`NULL + 'hello'::bytes`,
>
> Perhaps one of these should be <constant> + NULL (i.e. NULL on the right
> hand side).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2848/files#r42165171>.
>